tests/QAN/timerange_test.js

```diff
@@ -82,3 +84,116 @@ Scenario(
     qanOverview.verifySorting(1, 'desc');
   },
 );
+
+Scenario(
+  'PMM-T1138 - Verify QAN Copy Button for URL @qan',
+  async ({ I, adminPage, qanOverview }) => {
+    adminPage.applyTimeRange('Last 12 hours');
+
+    const dateTime = moment().format('x');
+
+    qanOverview.waitForOverviewLoaded();
+    qanOverview.selectRow(2);
+    I.click(qanOverview.buttons.copyButton);
+
+    const url = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+    const toTimeFromUrl = qanOverview.getToTimeFromUrl(url);
+
+    assert.ok(Math.abs(dateTime - toTimeFromUrl) < 30000, 'Difference between moment time and first copied time must be less then half of minute');
+
+    I.wait(30);
+    I.refreshPage();
+    qanOverview.waitForOverviewLoaded();
+    I.click(qanOverview.buttons.copyButton);
+
+    const url2 = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+    const toTimeFromUrl2 = qanOverview.getToTimeFromUrl(url2);
+
+    assert.ok(Math.abs(toTimeFromUrl - toTimeFromUrl2) < 60000, 'Difference between moment time and second copied time must be less then one minute');
```

Comment:
also, add a check about toTimeFromUrl2 should not be the same as toTimeFromUrl1, it must at least have a difference of 30 seconds.

Reply:
ok.thanks